### PR TITLE
Plugins

### PR DESCRIPTION
**Pull request by jkrol2**

### CODE REVIEW

**Comment by paulfantom** [comment]
@jkrol2 in tests don't check if anything is running on 0.0.0.0 it never works. Overwrite address in playbook and test for overwritten address, like 127.0.0.1.

**Comment by paulfantom** [comment]
@jkrol2 done? :stuck_out_tongue:

File: tasks/install.yml [hunk 1]
```diff
@@ -47,12 +47,30 @@
...
-- name: Install plugins
+- name: Set fluentd dirs ownership
+  file:
+    path: "{{ item }}"
+    owner: td-agent
+    group: td-agent
+    recurse: yes
+    state: directory
+  with_items:
+  - "/etc/td-agent"
```
Comment (paulfantom) [inline tasks/install.yml]: I don't think directories containing configuration files (in `/etc` dir) should be owned by a user which runs a program.

**Comment by jkrol2** [comment]
I think so :D

File: tasks/install.yml [hunk 2]
```diff
@@ -47,12 +47,30 @@
 
 - meta: flush_handlers
 
-- name: Install plugins
+- name: Set fluentd dirs ownership
```
Comment (paulfantom) [inline tasks/install.yml]: Why is this task in `install.yml` while [another task](https://github.com/cloudalchemy/ansible-fluentd/blob/f802b623315739517ae785e48b749fff3662fd4e/tasks/configure.yml#L2) which also sets file permissions is in `configure.yml`, couldn't it be unified?

File: tasks/install.yml [hunk 3]
```diff
@@ -47,12 +47,30 @@
 
 - meta: flush_handlers
```
Comment (paulfantom) [inline tasks/install.yml]: Why is it here? Handlers are usually executed at the end of a play, and flushing them earlier should be used only when necessary.

**Review by paulfantom — CHANGES_REQUESTED** [review]

File: tasks/install.yml [hunk 4]
```diff
@@ -74,3 +72,5 @@
     dest: /etc/td-agent/plugin
   with_items: "{{ fluentd_plugins }}"
   notify: restart td-agent
+
+- meta: flush_handlers
```
Comment (paulfantom) [inline tasks/install.yml]: Why???

Comment (jkrol2) [reply tasks/install.yml]: Well, in fact I've just left it as it was before. I'm not sure if it is essential - I will check it tomorrow

**Comment by paulfantom** [comment]
@jkrol2 travis says:
> [ANSIBLE0002] Trailing whitespace

Your darkest enemy :trollface:

**Comment by jkrol2** [comment]
My nemezis!

**Review by paulfantom — APPROVED** [review]

**Pull request merged**